sql/moz-fx-data-shared-prod/telemetry_derived/fx_health_ind_searches_by_provider_v1/metadata.yaml

```diff
@@ -0,0 +1,22 @@
+friendly_name: Fx Health Ind Searches By Provider
```

Comment:
What does the "ind" stand for?

Reply:
Good catch, I'll fix that in the metadata to put the full name - basically these are a set of aggregate tables all used in a dashboard called the "Firefox Health Indicator" Dashboard, so I was prefixing all of them the same to keep them together and putting them all in the same DAG